demo/helm/rook-operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: rook-operator
+version: 0.1.0
```

Comment:
I took @hunter's PR and made a stand alone helm chart out of it. https://github.com/jefflaplante/helm-rook-operator